src/CometFlexVoting.sol

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity >=0.8.10;
+
+import {Comet} from "comet/Comet.sol";
+import {CometConfiguration} from "comet/CometConfiguration.sol";
+import {Checkpoints} from "@openzeppelin/contracts/utils/Checkpoints.sol";
+
+import {FlexVotingClient} from "src/FlexVotingClient.sol";
+
+// TODO add description
```

Comment:
Should we go ahead and flesh out the description as part of this PR?

Reply:
Yes, thanks for flagging

Reply:
Done in c3b2fe3